routes/resultRouters.js

```diff
@@ -0,0 +1,21 @@
+const express = require('express');
+const resultController = require('../controllers/resultController');
```

Comment:
to keep it concestent let's make it
```
const { resultController } = require('../controllers');
```

and add the **resultController** to the ___./src/controllers/index.js___